src/main/java/teammates/logic/backdoor/BackDoorLogic.java

```diff
@@ -427,6 +435,9 @@ public void removeDataBundle(DataBundle dataBundle) {
             }
         }
         accountsDb.deleteAccounts(dataBundle.accounts.values());
+
+        // Delete all admin emails
```

Comment:
Comments on lines 427, 428 in the same method start with an upper case, so I simply followed for this line.